tools/cloud-build/daily-tests/tests/chrome-remote-desktop.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+test_name: chrome-remote-desktop
+deployment_name: chrome-remote-desktop-{{ build }}
+zone: us-central1-c
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/tools/validate_configs/test_configs/remote-desktop.yaml"
```

Comment:
I believe the test configs blueprints were a dumping ground for stuff that just literally has to pass validation.

I am wondering if we should enforce a duplicated file under tools/cloud-build/daily-tests/tests/ as that makes it clear this blueprint is used for an integration test and so future changes to it are less liberal.  What do you think?

Reply:
`tests/slurm-v5-startup-scripts.yml` & `tests/pbspro.yml` also reference a validation config. I would vote not to do it (duplicate the files) unless it becomes a problem.

Reply:
if there are precedents, then I'm ok with this. I would say I missed it in the past, but also not a huge deal since tests run every day.